### ISSUE TITLE
How to disable `trackingSessionEvent`?

### ISSUE THREAD

**Issue by scana** [amplitude/Amplitude-Kotlin#65]
## Summary
Hi! Is there any preferred way to disable `trackingSessionEvent` property on Android?
I see that Android-supported initialisation method only allows to modify the existing configuration, which in turn declares `trackingSessionEvent` as a `val`.

```kotlin
fun Amplitude(apiKey: String, context: Context, configs: Configuration.() -> Unit): com.amplitude.android.Amplitude {
    val config = Configuration(apiKey, context)
    configs.invoke(config)
    return com.amplitude.android.Amplitude(config)
}
```

**EDIT:**
Should I just do this?
```kotlin
val amplitude = Amplitude(Configuration(apiKey, context))
```

**Comment by qingzhuozhen** [comment]
Hi @scana, yeah, in the second way you can declare `trackingSessionEvents` to be false. 

@liuyang1520 @justin-fiedler maybe we should turn that value in config to var to prevent future confusion.

**Comment by liuyang1520** [comment]
@qingzhuozhen , do you mean [this part](https://github.com/amplitude/Amplitude-Kotlin/blob/f7b50d1c4d236b37b40fe86d409dce35253ae2f0/android/src/main/java/com/amplitude/android/Configuration.kt#L30-L38)?

**Comment by qingzhuozhen** [comment]
@liuyang1520 yeah, that is correct.

**Comment by liuyang1520** [comment]
@qingzhuozhen , I create a PR https://github.com/amplitude/Amplitude-Kotlin/pull/68 for updating this, PTAL!

**Comment by liuyang1520** [comment]
Merged and released in v1.3.0.
